test/blockchain_read_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 ProximaX Limited. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+package test
+
+import (
+	"github.com/proximax-storage/nem2-sdk-go/sdk"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/context"
+	"math/big"
+	"net/http"
+	"testing"
+)
+
+const (
+	iter     = 1000
+	pageSize = 32
+)
+
+func TestMosaicService_GetMosaicsFromNamespasceExt(t *testing.T) {
+
+	cfg, _ := sdk.NewConfig("http://190.216.224.11:3000", sdk.MijinTest)
```

Comment:
I assume this is an integration test?

Reply:
Yes, but it was, in fact, only a test sketch. It work and get some information about transaction but it don't check full functioanal